packages/guides/src/Interlink/Inventory.php

```diff
@@ -16,7 +18,7 @@ final class Inventory
 
     private bool $isLoaded = false;
 
-    public function __construct(private readonly string $baseUrl)
+    public function __construct(private readonly string $baseUrl, private readonly AnchorReducer $anchorReducer)
```

Comment:
Is it the responsibility of an Inventory to use the AnchorReducer? Isn't that part of the one consuming the links from the inventory group?

Reply:
Yes because by reducing the anchors before putting them into the inventory and after getting them from the inventory we enforce rules like that anchors are case insensitive and that results with minus and underscore are treated the same.

If this behaviour is not desired - if for example you want case-sensitive anchors - you could still pass a different AnchorReducer to the Inventories

Reply:
I renamed it into anchor reducer